standards/open-to-contributions.md

```diff
@@ -11,6 +11,7 @@ order: 4
 * The codebase MUST allow anyone to submit suggestions for changes to the codebase
 * The documentation MUST link to both the public issue tracker and submitted codebase changes
 * The codebase MUST include an email address for security issues and responsible disclosure
+* The codebase SHOULD include contribution guidelines explaining how contributors can get involved
```

Comment:
I think this is absolutely crucial, thanks for adding, perhaps we should even `MUST` this.